doc/ubuntu_release_process.md

```diff
@@ -156,6 +156,12 @@ build-now.
 This is generally needed when we are disabling backported feature from tip
 in order to retain existing behavior on an older series.
 
+**Note:** If adding a quilt patch to modify functionality introduced by
+a debian/patches/cpick-\* file, the new quilt patch needs the prefix
+fix-cpick-<previous_cpick_filename> because new-upstream-snapshot removes all
```

Comment:
I don't think this is quite accurate: `new-upstream-snapshot` removes the `cpick-*` and `fix-cpick-*` files for commits that are in the history of the git commit being snapshot, not "all" of them. (If it did remove all of them, then the naming wouldn't be important.)

Reply:
Tried rewriting this paragraph as follows:
```
**Note:** If adding a quilt patch to modify functionality introduced by
a debian/patches/cpick-\* file, the new quilt patch should have the prefix
fix-cpick-<the_ancestor_commitish_for_the_fixed_cpick>. `new-upstream-snapshot` removes all `cpick-<hash>-\*` and fix-cpick-<hash>-\*` files from the
`debian/patches` directory when the <hash> commit is already applied to the
commit history of the snapshot. 
```